doc/parameters/param_list.md

```diff
@@ -155,8 +155,11 @@ When `plugins` parameter is not overridden, the following default plugins are lo
 | ----------| --------| ------------|
 | controller_frequency | 20.0 | Frequency to run controller |
 | progress_checker_plugin | "progress_checker" | Check the progress of the robot |
```

Comment:
Should we explain what is meant by progress?

Reply:
Can you provide a suggested text?

Reply:
Plugin used by the controller to check whether the robot has at least covered a set distance/displacement in a set amount of time, thus checking the progress of the robot. What do you think?

Reply:
Done.